libs/eo/shared/atomic-design/ui-atoms/src/lib/eo-media/eo-media.presenter.ts

```diff
@@ -0,0 +1,146 @@
+/**
```

Comment:
Question: What is the reason for not looking into some kind of well proven grid system we can use throughout all types of components for aliigning content?

Reply:
Reason A: Watt already has a 12 column grid described as an overlay in Figma (currently disabled and only available to toggle by UX designers) but we haven't invested in implementing it. We might customize [Angular Material Grid list](https://material.angular.io/components/grid-list/overview) or implement one ourselves. It would need to support at least small and large screen configurations as well as fluid box sizes.
Reason B: Kenneth forgets to use the Watt grid in his designs.
Reason C: We don't need a grid on the landing page as we only have 1-2 element boxes.